example/android/app/build.gradle

```diff
@@ -15,7 +15,7 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 25
+    compileSdkVersion 28
     buildToolsVersion '25.0.3'
```

Comment:
Should we upgrade `buildToolsVersion` to v28 too? (or remove it so Android Studio will automatically pick the latest one)

Reply:
Yes we should remove or upgrade ```buildToolsVersion```.